db/migrate/20160304025530_add_overdue_count_to_equipment_models.rb

```diff
@@ -0,0 +1,19 @@
+class AddOverdueCountToEquipmentModels < ActiveRecord::Migration
+
+  def self.up
+    add_column :equipment_models, :overdue_count, :integer, :null => false, :default => 0
+
+    conn = ActiveRecord::Base.connection
+
+    conn.exec_query('select * from equipment_models').each do |e|
+      e.overdue_count = e.reservations.overdue.count
```

Comment:
We need to remove the model methods from this call as well; we want the migration to be entirely decoupled from our models.